cnd/cnd.ui/src/org/netbeans/modules/cnd/resources/templates/qt/widget.template

```diff
@@ -1,3 +1,23 @@
+<!--
```

Comment:
Are these user templates? If so, this looks as if the license header is taken over into the generated file. I would not add a header then.